bin/xud

```diff
@@ -23,6 +23,12 @@ const { argv } = require('yargs')
       describe: 'Path to the log file',
       type: 'string',
     },
+    network: {
+      describe: 'The network to use for lnd clients',
```

Comment:
Shouldn't it rather be `The network the lnd clients are using`? Because we are not telling the LNDs what they should do but just get information about what they are already doing on their own?

Reply:
Yup, that's definitely more accurate, I'll update it.